recipes/pysheds/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
please make sure to submit this license file to the upstream repo along with adding it to the manifest.in so that it ends up on pypi.

Reply:
LICENSE has been added to the package in this pull request: https://github.com/mdbartos/pysheds/pull/59